Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/BinaryExpression.cs

```diff
@@ -24,8 +24,11 @@ or SwitchExpressionSyntax
                     or SwitchStatementSyntax
                     or WhereClauseSyntax
                     or WhileStatementSyntax
-            || node.Parent is ConditionalExpressionSyntax
-                && node.Parent.Parent is not ArgumentSyntax;
+            || (
+                node.Parent is ConditionalExpressionSyntax conditionalExpressionSyntax
```

Comment:
I think you could have done some fancy pattern matching shenanigans like:
```
node.Parent is ConditionalExpressionSyntax
{ 
    WhenTrue: not node,
    WhenFalse: not node
}
```

Though, I don't know if it will allow `node` as the target or if it needs to be a literal value.